packages/@uppy/tus/src/index.js

```diff
@@ -279,25 +268,17 @@ module.exports = class Tus extends BasePlugin {
       this.uploaders[file.id] = upload
       this.uploaderEvents[file.id] = new EventTracker(this.uppy)
 
-      // Make `resume: true` work like it did in tus-js-client v1.
-      // TODO: Remove in @uppy/tus v2.
-      if (opts.resume) {
-        upload.findPreviousUploads().then((previousUploads) => {
-          const previousUpload = previousUploads[0]
-          if (previousUpload) {
-            this.uppy.log(`[Tus] Resuming upload of ${file.id} started at ${previousUpload.creationTime}`)
-            upload.resumeFromPreviousUpload(previousUpload)
-          }
-        })
-      }
+      upload.findPreviousUploads().then((previousUploads) => {
+        const previousUpload = previousUploads[0]
+        if (previousUpload) {
+          this.uppy.log(`[Tus] Resuming upload of ${file.id} started at ${previousUpload.creationTime}`)
+          upload.resumeFromPreviousUpload(previousUpload)
+        }
+      })
 
       let queuedRequest = this.requests.run(() => {
         if (!file.isPaused) {
-          // Ensure this gets scheduled to run _after_ `findPreviousUploads()` returns.
-          // TODO: Remove in @uppy/tus v2.
-          Promise.resolve().then(() => {
-            upload.start()
-          })
+          upload.start()
```

Comment:
Did you test this? will it work without the Promise.resolve() ? Maybe safer to put the remaining code inside the `then` callback of `upload.findPreviousUploads().then((previousUploads) => {`, or use async/await ?

Reply:
I tested it by checking if I can still successfully click pause/resume locally. I think this is fine. What do you think @goto-bus-stop?